components/announcement_bar/cloud_announcement_bar/user_limit_announcement_bar.tsx

```diff
@@ -67,7 +67,7 @@ export default class UserLimitAnnouncementBar extends React.PureComponent<Props>
 
     render() {
         const {userLimit, analytics, preferences} = this.props;
-
+        console.log(this.props);
```

Comment:
should this be removed?

Reply:
Yep. Thought I had a bug but didn't forgot to remove the log.